app/src/org/commcare/activities/EntitySelectActivity.java

```diff
@@ -852,16 +856,7 @@ public void deliverLoadResult(List<Entity<TreeReference>> entities,
                                   List<TreeReference> references,
                                   NodeEntityFactory factory, int focusTargetIndex) {
         loader = null;
-
-        AdapterView visibleView;
-        if (shortSelect.shouldBeLaidOutInGrid()) {

```

Comment:
Is this removing the gridview modality of entity selects entirely? I think it is mostly deprecated, but probably important to call out specifically if so to confirm that it's safe to do so.

Reply:
No, the intention here was to remove duplicattion with [setupUI()](https://github.com/dimagi/commcare-android/blob/dbbcfd48c9a5e7ddc447b494b5d8598e4abf1db8/app/src/org/commcare/activities/EntitySelectActivity.java#L262.) which gets called in Activity's `onCreate`

Reply:
aha, got it. I didn't see that code duplicated elsewhere so I assumed it was removed for being old/broken. Thanks for clarifying